src/extension.ts

```diff
@@ -85,13 +86,21 @@ function getServerOptions(config): ServerOptions
 function createClient(config: any): LanguageClient {
     let serverOptions = getServerOptions(config);
 
+    let {verbosity} = config.trace.server;
+    let trace = {
+        "off" : RevealOutputChannelOn.Never,
+        "message" : RevealOutputChannelOn.Warn,
+        "verbose" : RevealOutputChannelOn.Info
```

Comment:
is this the same feature? is there some documentation / comparable language server extension to compare with?

Reply:
intelliphense just disables it: https://github.com/bmewburn/vscode-intelephense/blob/master/src/extension.ts#L158